sgrep_lint/tests/run-perf-tests.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+set -e
+# set -x
+
+HYPERFINE="hyperfine -m 2 --warmup 0 "
```

Comment:
probably need to add this to build/test dependencies somewhere at some point.
Looks like a cool tool.